styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -1,42 +1,23 @@
-{% include '@atoms/skip-nav.twig' %}
+<div class="ama__layout--three-column">
+  {# Keeping tabindex for accessibility purposes #}
+  <article class="ama__page-content" role="article">
+    {% block pageContent %}
+      {% set placeholder = three_column.page_content.placeholder %}
+      {% include '@base/placeholder.twig' %}
+    {% endblock %}
+  </article>
 
-<div class="container">
-  {% block ribbon %}
-    {% set placeholder = three_column.ribbon.placeholder %}
-    {% include '@base/placeholder.twig' %}
-  {% endblock %}
+  <aside id="sidebar-primary" class="ama__sidebar-primary" role="complementary">
```

Comment:
Since "primary" is a modifer of "sidebar" the class should be `ama__sidebar--primary`